### PR TITLE
feat: added home page

### PR DESCRIPTION
**Pull request by sharath-io**
deploy preview:  https://deploy-preview-2--meta-store.netlify.app
Screen Shot 2022-02-20 at 6 48 35 PM

### CODE REVIEW

File: css/project.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.navigation {
+    background-color: var(--text-color) !important;
+    color: var(--secondary-color) !important;
+}
+
+.navigation .link {
+    color: var(--secondary-color) !important;
```
Comment (bharati-21) [inline css/project.css]: Kindly avoid styles with important. You can try using a higher specificity.

File: css/project.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    font-weight: bold;
+}
+
+.card-container {
+    height: 70rem;
+    padding: 1rem;
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    background-color: var(--text-color) !important;
```
Comment (bharati-21) [inline css/project.css]: Here as well.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (bharati-21) [inline index.html:7]: Not important, but you could add meta author and description tag.

**Review by bharati-21 — APPROVED** [review]
Kindly avoid `important` keyword for overriding styles.
Apart from that, looks good to be merged.

File: css/project.css (hunk 1 above)
Comment (sharath-io) [reply css/project.css]: Will look into this. Thank you for suggestions.

File: css/project.css (hunk 2 above)
Comment (sharath-io) [reply css/project.css]: Will update this.

File: index.html (hunk 3 above)
Comment (sharath-io) [reply index.html:7]: WIll add meta tag.

**Review by vidhi-thakur — APPROVED** [review]
agree with bharti's comments, other than that looks fine 👍

File: css/project.css (hunk 1 above)
Comment (sharath-io) [reply css/project.css]: Updated with specificity. It is very easy to use. Thanks for reminding

**Comment by sharath-io** [comment]
Updated  specificity.

**Pull request merged**